webapp/maps_heatmaps/test/clickInfoWindow.ts

```diff
@@ -0,0 +1,210 @@
+/*
```

Comment:
What is this file for? Is duplicated and should be removed, no?

Reply:
I removed it.